spec/cypress_on_rails/vcr/insert_eject_middleware_spec.rb

```
@@ -0,0 +1,167 @@
+              env['PATH_INFO'] = path
+
+              response = subject.call(env)
+
+              expect(response).to eq([200, {}, ["app did #{path}"]])
+              expect(vcr).to have_received(:turn_off!)
+            end
+          end
+        end
+      end
+    end
+  end
```

rubocop

[convention] Module has too many lines. [130/100]

(Metrics/ModuleLength)